app-office/moneydance/files/moneydance-bin

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# Use OpenJDK 11
+JAVA_HOME="/usr/lib/jvm/openjdk-11"
```

Comment:
ENODIR. You are still relying on masked java-vm behavior. I suppose you'll need to do some `/opt` globbing.

Reply:
Updated script, please have a look, if this is okay for you.